sky/skylet/job_lib.py

```diff
@@ -290,8 +294,9 @@ def get_latest_job_id() -> Optional[int]:
         return job_id
 
 
-def get_job_time_payload(job_id: int, is_end: bool) -> Optional[int]:
-    field = 'end_at' if is_end else 'start_at'
+def get_job_submitted_or_ended_timestamp_payload(
+        job_id: int, get_ended_time: bool) -> Optional[int]:
+    field = 'end_at' if get_ended_time else 'submitted_at'
```

Comment:
Can we add a comment saying something like: for a PENDING job, returning `submitted_at` is okay because the logic of displaying job duration in xxx function will check if `start_at` exists for this job (if not, it will not print any duration rather than using `submitted_at` as the starting point).

Or something better ;)

Reply:
Added the comment. Thanks!